Makefile

```diff
@@ -50,18 +50,23 @@ ruff-apply:
 	pipenv run ruff check --fix .
 
 # CLI commands
-shell:
-	pipenv run harvest-dockerized shell
+docker-shell:
```

Comment:
How/when is this command used vs. `test-harvest-local`?

Reply:
The Makefile command `test-harvest-local` actually kicks off a harvest, while this `docker-shell` command merely opens a bash shell in the a docker container of the application image.  It's mostly for debugging.

Reply:
It doesn't have to be done as part of this PR, but I think it would be helpful if the README had instructions on how to use this command for debugging!